README.md

```diff
@@ -1,13 +1,132 @@
-# HEPML Installer
+# HEPML Environment
 
-Installs a minimal Python3 environment for HEP machine learning
+Provides a minimal Python3 environment for machine learning focused towards HEP use cases
 
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 [![Build Status](https://travis-ci.com/matthewfeickert/HEPML-env.svg?branch=master)](https://travis-ci.com/matthewfeickert/HEPML-env)
 
+---
+
+Setting up a machine learning environment has gotten easier recently, but there are at times still problems that arise from time to time. HEPML-env allows for easily setting up a standard machine learning Python environment that should allow you to get to work with HEP data immediately. It should be machine agnostic, such that it can setup an identical environment on your laptop or on LXPLUS.
+
+### The default environment
```

Comment:
I admit there are some things on here that I've never used, but let's see how long it takes to build this full list on lx7.

Reply:
It is pretty quick to be honest. However, I can quantify that pretty easily with some timing scripts. :+1: